pkg/tuned/cmd/render/render.go

```diff
@@ -146,7 +148,7 @@ func render(inputDir []string, outputDir string, mcpName string) error {
 
 	if len(filteredPerformanceProfiles) == 0 {
 		klog.Warningf("Cannot find any PerformanceProfile applicable to MachineConfigPool %s.", mcpName)
-		return fmt.Errorf("Cannot find any PerformanceProfile complying with MachineConfigPool %s.", mcpName)
```

Comment:
I still do not see why this would be an error. In case no PP is present, there is still the openshift-node profile. The code from line 167 and below is going to work just fine.

This is a tuned renderer, why should it require a PP?